README.md

```diff
@@ -1 +1,64 @@
-# The-Countries
\ No newline at end of file
+# The Entire World
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Built With
+
+- React (CRA)
+- React Router Dom
+- Redux Toolkit
+- Bootstrap
+
+## sneak peeks
+
+![Application screenshot](./src/assets/Screenshot-dek.png)
+
+
+![Application screenshot](./src/assets/Screenshot-mob-home.png) ||  ![Application screenshot](./src/assets/Screenshot-mob-details.png)
+
+
+## Getting Started
+
+**Prerequisites:** A Web Browser (preferably FireFox or Chrome)
+To get a local copy up and running follow these simple exampl
+
+#### Live demo
+
+[The Entire World Website](http://ridwanediallo.github.io/The-Countries)
+
+
+#### Capston project video
+
+[Video Link](https://youtu.be/_oDtvPwq8io)
+
+### **Option 2**
+
+#### Set up your own copy locally
+
+- Clone [the GitHub Repository](https://github.com/ridwanediallo/The-Countries.git)
+- Go to the Project `cd "The-Countries"`
+- Run `npm install` to install the _dependencies_
+- Run `npm start` to run the live server.
+- Run `npm test` to run the tests
+
+## About me
+
+👤 **Ridwan**
+
+- GitHub: [@githubridwan](https://github.com/ridwanediallo)
+- Twitter: [@twitterridwan](https://twitter.com/RidwaneD)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ridwan-diallo)
+
+## Contributors
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Kindly add an Acknowledgements for the author of the design [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) as required by the Creative Commons License.